client/data/marketplace/use-wpcom-plugins-query.ts

```diff
@@ -37,6 +37,16 @@ const fetchWPCOMPlugins = ( type: Type, searchTerm?: string, tag?: string ) => {
 	);
 };
 
+export const getWPCOMPluginsQueryParams = (
+	type: Type,
+	searchTerm?: string,
+	tag?: string
+): [ QueryKey, QueryFunction< { results: Array< { [ plugin: string ]: Plugin } > } > ] => {
```

Comment:
The `Plugin` type is not defined. More precisely, it points to some obscure deprecated built-in DOM type for a "browser plugin", whatever that is, not to a Calypso-defined type.